config/routes.rb

```diff
@@ -111,9 +111,12 @@
     post "unhide"
     post "save"
     post "unsave"
-    get "suggest"
-    post "suggest", action: "submit_suggestions"
     post "disown"
+    resources :suggestions, only: [:new, :create]
+
+    # Mapping old routes to new routes. can be safely removed after the next deployment
+    get "suggest", to: "suggestions#new"
+    post "suggest", to: "suggestions#create"
```

Comment:
> Let’s use Rails’ resource routes for the new controllers, and be careful to route old URLs to the new actions so that people who have loaded old pages before the deploy don’t get errors as they submit/vote/hide/save/etc. (Note them with dated comments, we can delete them after a month or so.)

> add an explicit note: I don't want to maintain the old URL routes, I want use the Rails routes conventions for these.

I am little confused here. Should I remove these two line ?

Reply:
The above test fails with the following error. 
```
     ActiveRecord::RecordNotFound:
       Couldn't find Story with [WHERE `stories`.`short_id` = ?]
```
because it expect story_id in the url. like this. `http://localhost:3000/stories/d0xzoq/suggestions/new`

Not sure how to pass this. Sorry, I am new to Rspec and testing in rails.